app/scripts/views/download/download-controller.js

```diff
@@ -57,9 +57,18 @@
         function csvTransform(data) {
             var headers = _.keys(_.omit(data[0], '$$hashKey'));
 
+            var quoteReplaceRe = /"/g;
+
             var csvString = headers.join(',') + '\n';
             angular.forEach(data, function (row) {
-                csvString += _.values(_.omit(row, '$$hashKey')).join(',') + '\n';
+                csvString += _(row).omit('$$hashkey').values().map(function (value) {
+                    if (_.isString(value)) {
+                        // Need to quote strings, and convert double quotes to single quotes
+                        return '"' + value.replace(quoteReplaceRe, '\'') + '"';
```

Comment:
Correct. Double quotes are used by CSV to determine where each column value starts and ends, so if double quotes are left inside the string value the output CSV will be broken. Safest thing to do seemed to be to just swap all double quotes in the string to single quotes.